eng/pipelines/templates/steps/archetype-sdk-docs.yml

```diff
@@ -18,10 +18,6 @@ steps:
     - pwsh: |
```

Comment:
Can we please turn this into our multiple powershell task invocation since it is only calling one PS script.